### ISSUE TITLE
Duplicate rows 

### ISSUE THREAD

**Issue by cpavloud** [emo-bon/observatory-roskogo-crate#7]
In water logsheets, EMOBON_ROSKOGO_Wa_210618_3um_1 is appearing twice

**Comment by cpavloud** [comment]
In sampling and measured

**Comment by cpavloud** [comment]
@melinalou

**Comment by melinalou** [comment]
I can only find one appearance.

**Comment by cpavloud** [comment]
It's twice. 

Check [here ](https://github.com/emo-bon/observatory-roskogo-crate/blob/main/logsheets/transformed/water_measured.csv)
and [here ](https://github.com/emo-bon/observatory-roskogo-crate/blob/main/logsheets/transformed/water_sampling.csv)

**Comment by melinalou** [comment]
yes indeed..but you are looking in the transformed logsheets while I am working here https://docs.google.com/spreadsheets/d/1BCu2pbuIS4f-yrw5Kw4uzfO9v6ryZJBtSF5WBLWG_-E/edit?gid=15718907#gid=15718907. From our conversation with Katrina I concluded that this is where I supposed to make any changes. Am I wrong?

**Comment by cpavloud** [comment]
So how there are duplicates appearing in the transformed logsheets if they are not in the raw ones? This is strange.

**Comment by melinalou** [comment]
I have no idea..

**Comment by cpavloud** [comment]
@kmexter @melanthia can you keep an eye on this?

**Comment by kmexter** [comment]
It is because the transformed ones were made months ago (check out the date in GH) and we have not been able to reharvest and QC since Bram left - @marc-portier and co need to get that action working again. So that duplicate has been corrected since the transformed CSVs were made
but I did correct duplicate blank samples in the logsheet which I found when looking at them :-D

**Comment by cpavloud** [comment]
Ok! Could you let me know when you do the next harvest?

**Comment by kmexter** [comment]
@marc-portier take note - as this may happen while I am on vacation
FYI Marc we have to talk next week as there are now half a dozen changes to the QC and TTL that have to change in the etl action. I will be raising issues next week on the action's repo to record that all
